tests/unit/hive_metastore/test_mapping.py

```diff
@@ -254,15 +254,27 @@ def test_unskip_on_schema():
     )
 
 
-def test_unskip_missing_table(caplog):
+def test_unskip_missing_table():
```

Comment:
Let's pick this up too:

```suggestion
def test_unskip_missing_table() -> None:
```